.bazelrc

```diff
@@ -26,6 +26,9 @@ build:disable_hardening --features=-guards --copt=-DOT_DISABLE_HARDENING=1
 # Use LLD for linking
 build --features=use_lld
 
+# Enable link-time optimizations
+build --features=lto
```

Comment:
Does LTO make sense to enable for _all_ files right away? Mostly, I'm curious whether this should be opt-in or opt-out, since the optimizer might not make the best choices for maintaining a small footprint.